dsl/dsl_cuda/tmpl8_CUDA.py

```diff
@@ -126,7 +130,7 @@
     for (unsigned int t = i_step; t < (i_step + n_step); t++)
     {
     //***// This is the loop over nodes, which also should stay the same
-        for (unsigned int i_node = threadIdx.y; i_node < n_node; i_node+=blockDim.y)
```

Comment:
did you notice any perf difference with unsigned vs signed?

Reply:
The reason for change is the index not the unsigned vs signed. Is there any indication that unsigned performs better (or should be used here)?

Reply:
no sorry, I somehow misread the diff line

Reply:
np. this made me realize that I have to update the indexing as well. you can review but I have to push a commit still.